### PR TITLE
fix spelling of postpartum

### PR DESCRIPTION
**Pull request by markgoho**

### CODE REVIEW

File: lib/aca_entities/magi_medicaid/mitc/transformers/mitc_to/request.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ class Request < ::AcaEntities::Operations::Transforms::Transform
                   map 'prior_insurance_end_date', 'Prior Insurance End Date'
                   map 'is_pregnant', 'Applicant Pregnant Indicator'
                   map 'children_expected_count', 'Number of Children Expected'
-                  map 'is_in_post_partum_period', 'Applicant Post Partum Period Indicator'
+                  map 'is_in_postpartum_period', 'Applicant Postpartum Period Indicator'
```
Comment (saikumar9) [inline lib/aca_entities/magi_medicaid/mitc/transformers/mitc_to/request.rb:37]: _MAGI In The Cloud_ expects us to send the attribute name as 'Applicant Post Partum Period Indicator'. Please revert this change.

File: spec/support/magi_medicaid/mitc/eligibility_request.json [hunk 2]
```diff
@@ -1,134 +1,130 @@
...
+      "Medicare Entitlement Indicator": "Y",
+      "Incarceration Status": "Y",
+      "Lives In State": "Y",
+      "Claimed as Dependent by Person Not on Application": "Y",
+      "Applicant Attest Long Term Care": "Y",
+      "Has Insurance": "Y",
+      "State Health Benefits Through Public Employee": "Y",
+      "Prior Insurance": "Y",
+      "Applicant Pregnant Indicator": "Y",
+      "Applicant Postpartum Period Indicator": "N",
```
Comment (saikumar9) [inline spec/support/magi_medicaid/mitc/eligibility_request.json:19]: _MAGI In The Cloud_ expects us to send the attribute name as 'Applicant Post Partum Period Indicator'. Please revert this change.

File: spec/support/magi_medicaid/mitc/eligibility_request.json [hunk 3]
```diff
@@ -1,134 +1,130 @@
...
+      "Medicare Entitlement Indicator": "N",
+      "Incarceration Status": "N",
+      "Lives In State": "Y",
+      "Claimed as Dependent by Person Not on Application": "N",
+      "Applicant Attest Long Term Care": "N",
+      "Has Insurance": "N",
+      "State Health Benefits Through Public Employee": "N",
+      "Prior Insurance": "N",
+      "Applicant Pregnant Indicator": "N",
+      "Applicant Postpartum Period Indicator": "N",
```
Comment (saikumar9) [inline spec/support/magi_medicaid/mitc/eligibility_request.json:77]: _MAGI In The Cloud_ expects us to send the attribute name as 'Applicant Post Partum Period Indicator'. Please revert this change.

File: spec/aca_entities/magi_medicaid/mitc/transformers/mitc_to/request_spec.rb [hunk 4]
```diff
@@ -168,8 +168,8 @@
         expect(person[:'Applicant Pregnant Indicator']).to eq(person1[:is_pregnant])
         expect(person).to have_key(:'Number of Children Expected')
         expect(person[:'Number of Children Expected']).to eq(person1[:children_expected_count])
-        expect(person).to have_key(:'Applicant Post Partum Period Indicator')
-        expect(person[:'Applicant Post Partum Period Indicator']).to eq(person1[:is_in_post_partum_period])
+        expect(person).to have_key(:'Applicant Postpartum Period Indicator')
```
Comment (saikumar9) [inline spec/aca_entities/magi_medicaid/mitc/transformers/mitc_to/request_spec.rb:171]: Revert this change as well.

File: spec/aca_entities/magi_medicaid/mitc/transformers/mitc_to/request_spec.rb [hunk 5]
```diff
@@ -168,8 +168,8 @@
...
+        expect(person[:'Applicant Postpartum Period Indicator']).to eq(person1[:is_in_postpartum_period])
```
Comment (saikumar9) [inline spec/aca_entities/magi_medicaid/mitc/transformers/mitc_to/request_spec.rb:172]: Revert this change as well.

**Review by saikumar9 — CHANGES_REQUESTED** [review]
Please review the comments and make the required changes.

**Comment by markgoho** [comment]
@saikumar9 the spelling is incorrect, the answer is not to revert the change, it's to fix it upstream: pull request here: https://github.com/ideacrew/medicaid_eligibility/pull/4

**Pull request closed without merging**